Gir.toml

```diff
@@ -1390,6 +1390,11 @@ status = "generate"
         name = "selection_data"
         const = true
     [[object.function]]
+    name = "set"
+        [[object.function.parameter]]
+        name = "selection_data"
+        const = true
```

Comment:
While you're here, you could also ignore the getter and implement it with its correct name :)

Reply:
I.e. `fn get_data()` should call `gtk_selection_data_get_data_with_length()`